rpc/gnmi/gnmi-commit-confirmed.md

```diff
@@ -103,8 +117,20 @@ If the server is not waiting for confirmation or if the value doesn’t match th
 FAILED_PRECONDITION or INVALID_ARGUMENT error is returned respectively.
 
 ### 3.2.3 Cancel
+
 Cancellation can be issued by providing `CancelRequest` as action in the extension. The value of `id`
 should be equivalent to the id of the on-going commit.
 
 If the server is not waiting for cancellation or if the value doesn’t match the on-going commit
 then FAILED_PRECONDITION or INVALID_ARGUMENT error is returned respectively.
+
+### 3.2.4 Set Rollback Duration
+
+When a commit is on-going, a new rollback duration can be set by providing `SetRollbackDurationRequest`
```

Comment:
Could you make it clear if the existing rollback counter will be reset or if the new duration value value will be added on to the on-going counter ?

Reply:
done in 19cd325

Reply:
Perhaps a similar comment, but I think we need a bit more clarity here.

Can you please explicitly specify the "starting point" of the new timer (do we count the duration from the initial commit timestamp, or from 0 after receiving `SetRollbackDurationRequest`)?

Reply:
done in [0c1d8f4](https://github.com/openconfig/reference/pull/203/commits/0c1d8f4216e0f51b4994f0053305e7933696c35f)